src/docs/committers.md

```diff
@@ -40,6 +40,41 @@ It is the responsibility of a Committer to ensure that the ongoing good health a
 
 Committers should respond to public requests for code reviews.  While it is not necessary for every review to be approved by every committer, the health of the community depends on quality vetting of incoming changes.  If only a single committer is tagged on a review and is unable to participate, the committer should respond to the review and recommend another committer.
 
+Contentious Decisions
+---------------------
+
+To help address cases where consensus cannot be reached even after extended discussion, committers
+may use a vote to reach a conclusion.
+
+Before calling a vote, it's very important to attempt to reach consensus without a vote. Because
+discussion and collaboration help us to understand one another's concerns and weigh them, issues
+that are potentially contentious generally deserve a thread on <pants-devel@googlegroups.com>: if
+you are unsure of whether an issue is contentious, consider sending the mail anyway.
+
+If it becomes clear that all concerns have been voiced, but that consensus cannot be reached via
+discussion, a committer may call a vote by creating a new thread on <pants-devel@googlegroups.com>
```

Comment:
Could we make it more clear here that this step is to be taken *only* in cases which have previously received discussion on pants-devel? The current reading seems like it could be interpreted as "making the initial thread on pants-devel is optional, but you can call a vote either way".